website/docs/docs/dbt-versions/2023-release-notes.md

```diff
@@ -0,0 +1,998 @@
+
+  </expandable>
+
+- <expandable alt_header='Docs updates'>
+
+    Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! We want to share some highlights introduced to docs.getdbt.com in the last month:
+
+    ## 🔎 Discoverability
+
+    - [API docs](/docs/dbt-cloud-apis/overview) now live in the left sidebar to improve discoverability.
+    - [The deploy dbt jobs sidebar](/docs/deploy/deployments) has had a glow up 💅 that splits the ‘about deployment’ into two paths (deploy w dbt cloud and deploy w other tools), adds more info about the dbt cloud scheduler, its features, and how to create a job, adds ADF deployment guidance. We hope the changes improve the user experience and provide users with guidance when deploying with other tools.
+
+    ## ☁ Cloud projects
+
+    - Added Starburst/Trino adapter docs, including:
+  * [dbt Cloud quickstart guide](/guides/starburst-galaxy), 
+  * [connection page](/docs/cloud/connect-data-platform/connect-starburst-trino), 
+  * [set up page](/docs/core/connect-data-platform/trino-setup), and [config page](/reference/resource-configs/trino-configs). 
+    - Enhanced [dbt Cloud jobs page](/docs/deploy/jobs) and section to include conceptual info on the queue time, improvements made around it, and about failed jobs. 
+    - Check out the April dbt [Cloud release notes](/docs/dbt-versions/dbt-cloud-release-notes)
+
+    ## 🎯 Core projects 
+
+    - Clearer descriptions in the [Jinja functions page](/reference/dbt-jinja-functions), that improve content for each card. 
+    - [1.5 Docs](/docs/dbt-versions/core-upgrade/upgrading-to-v1.5) have been released as an RC! 
+    - See the beautiful [work captured in Core v 1.5](https://github.com/dbt-labs/docs.getdbt.com/issues?q=is%3Aissue+label%3A%22dbt-core+v1.5%22+is%3Aclosed).
+
+    ## New 📚 Guides and ✏️ blog posts
+
+    - [Use Databricks workflows to run dbt Cloud jobs](/guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs)
+    - [Refresh Tableau workbook with extracts after a job finishes](/guides/zapier-refresh-tableau-workbook)
+    - [dbt Python Snowpark workshop/tutorial](/guides/dbt-python-snowpark)
+    - [How to optimize and troubleshoot dbt Models on Databricks](/guides/optimize-dbt-models-on-databricks)
+    - [The missing guide to debug() in dbt](/blog/guide-to-jinja-debug)
+    - [dbt Squared: Leveraging dbt Core and dbt Cloud together at scale](/blog/dbt-squared)
+    - [Audit_helper in dbt: Bringing data auditing to a higher level](/blog/audit-helper-for-migration)
+
+  </expandable>
+
+## March 2023
+
```

Comment:
it looks like march is missing the follow release notes:
- [trino public preview](https://github.com/dbt-labs/docs.getdbt.com/pull/5510/files#diff-f46466fceaef4fc26492b8f698890730c221ce5d2d7e19baf9bbac8c13b493f8L2-L7)
- [api v2 limits](https://github.com/dbt-labs/docs.getdbt.com/pull/5510/files#diff-172628a6624d99e369de4f764b46a6ea257a8df4beb50d36082d7802f681b7c3L2-L6)